offload/test/offloading/ompx_bare_multi_dim.cpp

```diff
@@ -0,0 +1,56 @@
+// RUN: env LIBOMPTARGET_INFO=63 %libomptarget-compilexx-run-and-check-generic
```

Comment:
Are you sure that this env propagates to the `run` stage of the test too?

I think this is why I see an error when running the botconfig. I do not see an error when I run the binary manually. Other tests appear to split this command into two. Example from `test/offloading/force-usm.cpp`

```
// clang-format off
// RUN: %libomptarget-compilexx-generic
// RUN: env LIBOMPTARGET_INFO=32 %libomptarget-run-generic 2>&1 | %fcheck-generic --check-prefix=NO-USM
```

Reply:
That's a really weird limitation...